packages/suite/src/components/suite/CoinjoinStatusBar.tsx

```diff
@@ -27,15 +27,23 @@ import { Translation } from './Translation';
 
 const SPACING = 6;
 
-const Container = styled.div`
+const Container = styled.div<{ isClickable: boolean }>`
     display: flex;
+    justify-content: space-between;
     align-items: center;
     height: 28px;
     padding: 0 ${SPACING}px;
     background: ${({ theme }) => theme.BG_WHITE};
     border-bottom: 1px solid ${({ theme }) => theme.STROKE_LIGHT_GREY};
     font-size: ${variables.FONT_SIZE.TINY};
     font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    ${({ isClickable, theme }) =>
+        isClickable &&
+        `
```

Comment:
This is the [recommended](https://styled-components.com/docs/api#css) way to interpolate + [code highlight](https://marketplace.visualstudio.com/items?itemName=styled-components.vscode-styled-components) starts working :)

```suggestion
       css`
```